musicbrainzngs/musicbrainz.py

```diff
@@ -685,7 +685,7 @@ def _get_auth_type(entity, id, includes):
     """ Some calls require authentication. This returns
     True if a call does, False otherwise
     """
-    if "user-tags" in includes or "user-ratings" in includes:
+    if "user-tags" in includes or "user-ratings" in includes or "user-genres" in includes:
```

Comment:
Now that we have 3 items to check here maybe we should have an `AUTH_REQUIRED_INCLUDES` set, and check the intersection of that with `includes` here instead of using magic numbers.

Reply:
We have no tests to see that

* we correctly raise an exception if one of these includes is set and the user hasn't authenticated.
* we correctly send the auth header if one of these includes is set and the user has authenticated

It'd be nice to add these

Reply:
Do you think `AUTH_REQUIRED_INCLUDES` should be a global instead of just a variable inside the given function?

Reply:
yes, I think this can be a global